src/control/client/config.go

```diff
@@ -134,18 +134,14 @@ func (c *Configuration) LoadConfig() error {
 	return nil
 }
 
-// ProcessEnvOverrides examines environment variables and applies the environment
-// variable value over an existing value if defined.
-func (c *Configuration) ProcessEnvOverrides() int {
-	var envVarsFound = 0
-
-	daosAgentDrpcSock := c.ext.getenv(daosAgentDrpcSockEnv)
-	if daosAgentDrpcSock != "" {
-		log.Debugf("DAOS_AGENT_DRPC_DIR found: %s", daosAgentDrpcSock)
-		c.RuntimeDir = daosAgentDrpcSock
-		envVarsFound++
+// ValidateEnv reads the value of a given environment variable and compares it with
+// the supplied value.  Issues a non-fatal debug message if there is a mismatch
+func (c *Configuration) ValidateEnv(envVar string, value string) error {
+	temp := c.ext.getenv(envVar)
+	if temp != value {
+		log.Debugf("Environment variable '%s' has value '%s' which does not match '%s'", envVar, temp, value)
 	}
-	return envVarsFound
+	return nil
```

Comment:
If it doesn't return anything except nil, should probably just return nothing

Reply:
Fair enough.

Reply:
in fact you don't even need this method, just compare c.ext.getenv(<sock_path>) == <cli_sock_path> where you call this function